application/src/main/java/org/thingsboard/server/controller/AlarmController.java

```diff
@@ -54,8 +55,7 @@
     public static final String ALARM_ID = "alarmId";
 
     @PreAuthorize("hasAnyAuthority('SYS_ADMIN', 'TENANT_ADMIN', 'CUSTOMER_USER')")
-    @RequestMapping(value = "/alarm/{alarmId}", method = RequestMethod.GET)
-    @ResponseBody
```

Comment:
Are you sure the @ResponseBody annotation may be safely removed here? Have you done any tests?

Reply:
@RestController is a specialized version of the controller. It includes the @Controller and @ResponseBody annotations, and as a result, simplifies the controller implementation:
ref: https://www.baeldung.com/spring-controller-vs-restcontroller

Reply:
Indeed. The issue with this PR is that you have chosen the controller that is not covered with integration tests for REST API. The changes are ok in general. Please, create a similar PR for the DeviceController. Once tests will pass, we will merge DeviceController PR and this one. Then you/we can start refactoring other controllers.

Reply:
Sure, Here is the PR https://github.com/thingsboard/thingsboard/pull/4881

Reply:
<img width="1661" alt="Screenshot 2021-07-09 at 7 05 25 AM" src="https://user-images.githubusercontent.com/11393142/125010519-4e5db100-e084-11eb-8a12-a9d7d460320c.png">

Reply:
@ashvayka Do you have any concerns about this? Are we good to go with the PR? Let me know your concerns, please.

Reply:
Any feedback / concerns on this PR please.

thank you!